testgrid/specs/deploy.yaml

```diff
@@ -37,7 +37,7 @@
       version: latest
   unsupportedOSIDs:
   - ubuntu-2204 # docker 19.x is not available on ubuntu 22.04
-  - rocky-91 # docker is not supported on rhel 9 variants
```

Comment:
wouldnt it be best to append rocky-92 to the array in all cases rather than replace rocky-91? in case we decide to add back 9.1 to any of the suites

Reply:
If we decide to add back 9.1 then we can just find replace in all project by - rocky-91 and replace by : - rocky-91 \n - rocky-92. It is very low effort, I think the effort so we do not need to have a data/info that is unused.